--- src/base/basictypes.h.broken	2012-02-02 21:36:23.000000000 +0000
+++ src/base/basictypes.h	2013-02-12 02:49:17.000000000 +0000
@@ -331,13 +331,12 @@
 
 #endif  // HAVE___ATTRIBUTE__ and __ELF__ or __MACH__
 
-#if defined(HAVE___ATTRIBUTE__) && (defined(__i386__) || defined(__x86_64__))
+#if defined(HAVE___ATTRIBUTE__) && (defined(__i386__) || defined(__x86_64__) || defined(__arm__) )
 # define CACHELINE_ALIGNED __attribute__((aligned(64)))
 #else
 # define CACHELINE_ALIGNED
 #endif  // defined(HAVE___ATTRIBUTE__) && (__i386__ || __x86_64__)
 
-
 // The following enum should be used only as a constructor argument to indicate
 // that the variable has static storage class, and that the constructor should
 // do nothing to its state.  It indicates to the reader that it is legal to
